coverage/docs/design.md

```diff
@@ -0,0 +1,139 @@
+#Overview
+This code coverage tool has two major features.
+1. In post-submit workflow (when a pull request is merged), it runs code coverage and generates the 
```

Comment:
This makes it sound like the post-submit workflow is associated with a PR which is not the case. The workflow is associated with a commit to the base branch so it can be run on any merged commit, not just when a pull request is merged.

Reply:
Done.